src/azure-cli/azure/cli/command_modules/netappfiles/tests/latest/test_volume_commands.py

```diff
@@ -84,21 +96,75 @@ def test_create_volume_with_subnet_in_different_rg(self):
 
         subnet_rg = self.create_random_name(prefix='cli-rg-', length=24)
         subs_id = self.current_subscription()
-        self.cmd("az group create -n %s --subscription %s -l %s" % (subnet_rg, subs_id, LOCATION)).get_output_in_json()
+        self.cmd("az group create -n %s --subscription %s -l %s" % (subnet_rg, subs_id, RG_LOCATION)).get_output_in_json()
 
         rg = '{rg}'
-        self.setup_vnet(subnet_rg, vnet_name, subnet_name, '10.0.0.0')
-        self.cmd("az netappfiles account create -g %s -a %s -l %s" % (rg, account_name, LOCATION)).get_output_in_json()
-        self.cmd("az netappfiles pool create -g %s -a %s -p %s -l %s %s" % (rg, account_name, pool_name, LOCATION, POOL_DEFAULT)).get_output_in_json()
+        self.setup_vnet(subnet_rg, vnet_name, subnet_name, '10.0.0.0', RG_LOCATION)
+        self.cmd("az netappfiles account create -g %s -a %s -l %s" % (rg, account_name, ANF_LOCATION)).get_output_in_json()
```

Comment:
For all your tests, can we add some checks or assert for command results?

Reply:
I suggest we do this change for next version/update that will come very soon.